### PR TITLE
pep8, rst fixes, doc-style, removal of Zope2.10 specialities

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

**Comment by gforcada** [comment]
I though that http://jenkins.plone.org/view/Pkgs/job/package-plone.transformchain/ would help :)

File: plone/transformchain/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (gforcada) [inline plone/transformchain/__init__.py]: Just FYI but this is actually not needed, if the file is completely empty it is fine to leave it like that :-) at least flake8 will not complain

**Comment by jensens** [comment]
@gforcada a great tiny helper :)

File: plone/transformchain/__init__.py (hunk 1 above)
Comment (jensens) [inline plone/transformchain/__init__.py]: Well, i usually like to have it in there, even if superfluous now, but what if someone starts adding code there?

Comment (gforcada) [inline plone/transformchain/__init__.py]: Fair point. I usually only do whatever flake8 forces me, nothing more nothing less :-)

**Pull request merged**